src/sentry/snuba/metrics/fields/snql.py

```diff
@@ -980,6 +980,47 @@ def on_demand_apdex_snql_factory(
     )
 
 
+def on_demand_count_miserable_factory(
```

Comment:
I need to redo this. No where near what it should be (this is from when I was trying to implement user_misery)